tests/foreman/ui/test_computeresource_rhev.py

```diff
@@ -1067,11 +1081,11 @@ def test_positive_provision_rhev_with_custom_compute_settings(self):
                     dict(
                         size='10',
                         storage_domain=self.rhev_storage_domain,
-                        bootable=False,
-                        preallocate_disk=True
+                        bootable=True,
```

Comment:
what is a reason to change values to opposite ones?

Reply:
One disk should be bootable and creating preallocated disk is just time consuming